src/workerd/jsg/dom-exception.h

```diff
@@ -109,17 +93,12 @@ class DOMException: public Object {
   void visitForMemoryInfo(MemoryTracker& tracker) const {
     tracker.trackField("message", message);
     tracker.trackField("name", name);
-    tracker.trackField("errorForStack", errorForStack);
   }
 
 private:
   kj::String message;
   kj::String name;
 
-  // We implement the `stack` property in a similarly hacky way as Chrome: store an Error object
```

Comment:
If this is good enough for Chrome, why isn't it good enough for us? I'm nervous about adding new patches to V8, don't think it's really worth it just to avoid an allocation on the error path. Though if V8 would accept upstreaming the patch, then maybe it makes sense?

Reply:
I've already got an open thread in the v8-dev group to ask if they would accept this change. Given how convoluted chrome's approach to handling this is, even compared to ours, I would have to imagine that it would be of benefit there also.